docs/source/checks/tabular/model_evaluation/plot_train_test_performance.py

```diff
@@ -21,7 +21,7 @@
 Scorers are a convention of sklearn to evaluate a model,
 it is a function which accepts (model, X, y_true) and returns a float result which is the score.
 A sklearn convention is that higher scores are better than lower scores. For additional details `see scorers
-documentation <https://scikit-learn.org/stable/modules/model_evaluation.html#scoring>`__.
+documentation <https://scikit-learn.org/stable/modules/model_evaluation.html#scoring>`_.
```

Comment:
```suggestion
documentation <https://scikit-learn.org/stable/modules/model_evaluation.html#scoring>`__.
```

two underlines is for an anonymous link (see [rst documentation](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks) and [stack overflow](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)), which is better since otherwise we may have conflicts (since we're currently using this in various places but redefining it every time)

Reply:
did not know that. thanks